pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -568,9 +575,12 @@ stdenv.mkDerivation (
       # Ffmpeg > 4 doesn't know about the flag anymore
       (enableFeature buildAvresample "avresample")
     ]
+    ++ optionals (lib.versionOlder version "8.0") [
+      # FFMpeg > 8 doesn't know about the flag anymore
+      (enableFeature (buildPostproc && withGPL) "postproc")
```

Comment:
Swapping the order of this list makes it so that all ffmpeg versions need to rebuild.

Reply:
No, move the condition back in between `buildAvutil` and `buildSwresample`

Reply:
https://github.com/pyrox0/nixpkgs/compare/init/ffmpeg-8...jopejoe1:ffmpeg-8-fixup?expand=1

Reply:
Thank you! merged this into my branch to ensure it retains attribution.